### PR TITLE
Switch from Jade to Pug

### LINKED ISSUE #12: Switch from Jade to Pug
> _"Jade has been renamed to pug, please install the latest version of pug instead of jade"_

[1] [./package.json:17](https://github.com/mozilla-magnet/magnet-adaptor-sched/blob/61b5903b9852e5eaa8dffe2585102df0cefcc401/package.json#L17)
[2] https://www.npmjs.com/package/pug
### Steps to reproduce:

``` sh
$ nvm use 4
Now using node v4.4.2 (npm v2.15.0)

$ npm i
npm WARN deprecated jade@1.11.0: Jade has been renamed to pug, please install the latest version of pug instead of jade
```

### PR DESCRIPTION
**Pull request by pdehaan**
Fixes #12

### CODE REVIEW

**Comment by arcturus** [comment]
Could you rebase?

File: lib/sched.js [hunk 1]
```diff
@@ -82,25 +82,24 @@ Venue.prototype.parseLight = function() {
...
+        var container = $(evt.children()[0]);
+        container.find('a').toArray().forEach((a) => {
+          a = $(a);
+          var link = this.getLink(a.attr('href'));
+          var title = a[0].children[0].data;
+
+          events.push({
+            time,
+            title,
+            link
```
Comment (pdehaan) [inline lib/sched.js:100]: Indenting was off by 2 spaces here (so sayeth ESLint).

**Comment by pdehaan** [comment]
Rebased.

**Comment by arcturus** [comment]
Thanks!

**Pull request merged**